src/cljs/witan/ui/controllers/datastore.cljs

```diff
@@ -863,11 +869,17 @@
 (defmethod on-activity-finished
   [:create-datapack :failed]
   [{:keys [args]}]
+  (log/warn "Datapack create failure: " (get-in args [:message :kixi.comms.event/payload]))
   (data/swap-app-state! :app/create-datapack assoc :cdp/pending? false)
-  (data/swap-app-state!
-   :app/create-datapack assoc :cdp/error
-   {:gefeature/neral (case (get-in args [:message :kixi.comms.event/payload :reason])
-                       :metadata-invalid (get-string :string/create-datapack-fail-invalid))}))
+  (data/swap-app-state! :app/create-datapack assoc :cdp/error
+                        (case (get-in args [:message :kixi.comms.event/payload :reason])
+                          :metadata-invalid :string/create-datapack-fail-invalid))
+  (comment
```

Comment:
`(comment ,,,)` returns nil. Is that what you want here?

Reply:
All these types of fn's are state changers, return values don't matter.